core/src/i18n/en.yaml

```diff
@@ -244,14 +244,14 @@ cluster-roles:
 clusters:
   add:
     errors:
-      failed-to-add-cluster: Failed to add a cluster
-    title: Add a Cluster
+      failed-to-add-cluster: Failed to connect a cluster
```

Comment:
Probably a question to @grego952 but shouldn't the word `Cluster` start upper-case?

Reply:
`Cluster` does not need to start with upper-case. `Failed to connect a cluster` is ok.